tools/github_actions/Dockerfile.template

```diff
@@ -96,11 +96,11 @@ RUN apt-get update \
     libxml2-dev libxslt1-dev libpq-dev \
     {extra_packages}
 
-RUN groupadd --gid 3434 circleci \
-  && useradd --uid 3434 --gid circleci --shell /bin/bash --create-home circleci \
-  && echo 'circleci ALL = (ALL) NOPASSWD: ALL' >> /etc/sudoers.d/50-circleci \
+RUN groupadd --gid 3434 github \
+  && useradd --uid 3434 --gid github --shell /bin/bash --create-home github \
+  && echo 'github ALL = (ALL) NOPASSWD: ALL' >> /etc/sudoers.d/50-github \
   && echo 'Defaults    env_keep += "DEBIAN_FRONTEND"' >> /etc/sudoers.d/env_keep
 
-USER circleci
+USER github
```

Comment:
We should username a variable at the top of the file so it only needs to change value once?